index.js

```diff
@@ -6,11 +6,15 @@ var YAML = require('yamljs');
 /**
  *
  * @param {number|string} performanceYear - four digit year
- * @return {Array<Benchmark>} - benchmarks data for given year
+ * @return {Array<Benchmark>|undefined} - benchmarks data for given year
  */
 exports.getBenchmarksData = function(performanceYear) {
-  return JSON.parse(
-    fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  try {
+    return JSON.parse(
+      fs.readFileSync(path.join(__dirname, 'benchmarks', performanceYear + '.json')));
+  } catch (error) {
+    return;
```

Comment:
Not sure if I should check the error code here. Other than the file not existing, I'm not sure what errors exist and if/how they should be handled.